editors/code/src/installation/server.ts

```diff
@@ -108,6 +110,8 @@ function isBinaryAvailable(binaryPath: string): boolean {
     console.log("Checked binary availablity via --version", res);
```

Comment:
Maybe we can stop logging to console if we add that command?

Reply:
Keeping it leaves all information necessary for debugging in one place.

Reply:
Yeah, but I feel we should log less stuff to console, possibly in favor of the output channel.

Reply:
I don’t have strong opinion about it.

Reply:
FYI just created an issue about logging #3250